interfaces/builtin/cuda_support.go

```diff
@@ -0,0 +1,34 @@
+package builtin
+
+const cudaSupportConnectedPlugAppArmor = `
+# Description: CUDA requires being able to read/write to some nvidia devices
+
+@{PROC}/sys/vm/mmap_min_addr r,
+@{PROC}/devices r,
+@{PROC}/modules r,
+@{PROC}/driver/nvidia/params r,
+/sys/devices/system/memory/block_size_bytes r,
+unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
+/dev/nvidia-uvm wr,
+/dev/nvidiactl wr,
+/{dev,run}/shm/cuda.* rw,
+`
+
+const cudaSupportConnectedPlugSecComp = `
+# Description: allow running operations on GPU devices
+# necessary as cuda tries to create /dev/nvidia-uvm-tools with mknod
+mknod - |S_IFCHR
```

Comment:
Isn't this created automatically by udev when nvidia-uvm gets loaded?

Reply:
It is, but CUDA still attempts to create it when running, which is why I think it's okay to just let it try and fail to create this if we don't want to grant this permission to the interface